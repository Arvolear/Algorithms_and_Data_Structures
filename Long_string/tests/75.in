57540587906412837423028091688255001
7096072192666929329255250902122