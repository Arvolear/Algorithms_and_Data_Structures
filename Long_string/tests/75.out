57547683978605504352357346939157123
57533491834220170493698836437352879
408312165792403139806743297295120980000020219138199547997228012122
8108